tensorflow/contrib/tensorrt/resources/trt_allocator.cc

```diff
@@ -27,7 +27,7 @@ namespace tensorflow {
 namespace tensorrt {
 
 // std::align is not supported, so this method mimic its behavior.
-void* Align(size_t alignment, size_t size, void*& ptr, size_t& space) {
+void* Align(int64_t alignment, int64_t size, void*& ptr, int64_t& space) {
```

Comment:
Do all of these need to be signed?

Reply:
Sorry I didn't noticed that size_t is unsigned, now I changed them to uint64_t.

Reply:
hmm, I thought the idea here was to actually use signed for some of these. If unsigned is fine, why not keeping `size_t`? In 64-bit platform, `size_t` is the same `uint64_t` as far as I know.

Reply:
I hard coded uint64_t because the trt IGpuAllocator API uses that: https://docs.nvidia.com/deeplearning/sdk/tensorrt-api/c_api/classnvinfer1_1_1_i_gpu_allocator.html

Reply:
I see. Since TRT doesn't support 32-bit platforms anymore, this should be fine.